src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -796,5 +980,95 @@ public async Task<string> GetNextRoatpApplicationReference()
+                        new
+                        {
+                            applicationId,
+                            applyData,
+                            grade = financialReviewDetails.SelectedGrade
+                        });
+                    if (recordsAffected > 0)
+                    {
+                        await connection.ExecuteAsync("UPDATE Apply SET FinancialReviewStatus = @financialReviewStatus WHERE ApplicationId = @applicationId",
+                            new
+                            {
+                                applicationId,
+                                financialReviewStatus
+                            });
+                    }
+                }
+                catch (Exception exception)
+                {
+                    _logger.LogError($"Unable to record financial grade for application {applicationId}", exception);
+                    return await Task.FromResult(false);
+                }
+            }
+
+            return await Task.FromResult(true);
+        }
+
+        public async Task<bool> StartAssessorReview(Guid applicationId, string reviewer)
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                await connection.ExecuteAsync(@"UPDATE Apply SET AssessorReviewStatus = @assessorReviewStatus,
+                                                UpdatedAt = @updatedAt, UpdatedBy = @updatedBy
+                                                WHERE ApplicationId = @applicationId
+                                                AND AssessorReviewStatus IN ( @draftStatus, @newStatus )",
+                        new
+                        {
+                            applicationId,
+                            assessorReviewStatus = AssessorReviewStatus.InProgress,
+                            updatedAt = DateTime.UtcNow,
+                            updatedBy = reviewer,
+                            draftStatus = AssessorReviewStatus.Draft,
+                            newStatus = AssessorReviewStatus.New
+                        });
+            }
+
+            return await Task.FromResult(true);
+        }
+
+        public async Task<bool> StartFinancialReview(Guid applicationId, string reviewer)
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                await connection.ExecuteAsync(@"UPDATE Apply SET FinancialReviewStatus = @financialReviewStatus,
+                                                UpdatedAt = @updatedAt, UpdatedBy = @updatedBy
+                                                WHERE ApplicationId = @applicationId
+                                                AND FinancialReviewStatus IN ( @draftStatus, @newStatus )",
+                        new
+                        {
+                            applicationId,
+                            financialReviewStatus = FinancialReviewStatus.InProgress,
```

Comment:
could this say financialReviewStatusInProgress so it's slightly more clear on what the status is being set to?

Reply:
Done, and the start assessor review status variable too